jupyterhub_moss/form/option_form.js

```diff
@@ -210,7 +210,14 @@ document.addEventListener("DOMContentLoaded", () => {
 
   // Reset when returning to simple tab
   document.getElementById('simple_tab_link').addEventListener(
-    'click', resetSpawnForm
```

Comment:
Why not simply removing this listener and let the HTML form keep its state in memory ?

Reply:
Because what is read by the server are the advanced tab fields.
The simple tab fields update the advanced tab ones, so you really want things to be sync, thus the reset when switching back from advanced to simple.

Reply:
Hm, I see. We should perhaps rethink completely the handling of the form state.

Reply:
Yes, I agree, this is hackish.